src/main/kotlin/org/rust/ide/sdk/flavors/RsSdkFlavor.kt

```diff
@@ -0,0 +1,59 @@
+/*
+ * Use of this source code is governed by the MIT license that can be
+ * found in the LICENSE file.
+ */
+
+package org.rust.ide.sdk.flavors
+
+import com.intellij.openapi.extensions.ExtensionPointName
+import com.intellij.openapi.util.SystemInfo
+import com.intellij.util.io.isDirectory
+import org.rust.cargo.toolchain.tools.Cargo
+import org.rust.cargo.toolchain.tools.Rustc
+import org.rust.stdext.isExecutable
+import java.nio.file.Path
+
+fun RsSdkFlavor.suggestHomePaths(): List<Path> =
```

Comment:
nit: Why this method is extension one and all other (e.g. `getApplicableFlavors`) are in companion object?
Let's place it in companion object as well